test/jdk/javax/swing/ImageIcon/ImageIconTest.java

```diff
@@ -39,63 +39,56 @@
 public class ImageIconTest {
 
     static enum ArgType { FILE, URL, BYTEARRAY, IMAGE };
+    static enum ArgVal { NULL, INVALIDDATA };
```

Comment:
Maybe name this `INVALID_DATA`:
```suggestion
    static enum ArgVal { NULL, INVALID_DATA };
```

Reply:
ok